client/packages/common/src/intl/locales/en/common.json

```diff
@@ -44,6 +44,10 @@
   "button.choose-file": "Choose file to upload",
   "button.clear": "Clear",
   "button.close": "Close",
+  "label.version": "Version",
+  "label.is_active": "Is active",
+  "label.is_custom": "Is custom",
```

Comment:
Yea first thing in mind that these might be too developery terms. Maybe can go without `is`?